packages/flutter_tools/lib/src/fuchsia/fuchsia_sdk.dart

```diff
@@ -55,7 +55,7 @@ class FuchsiaSdk {
   }
 
   /// Returns the fuchsia system logs for an attached device.
-  Stream<String> syslogs(String id) {
+  Stream<String> syslogs(String deviceIp) {
```

Comment:
Changing the parameter name might lead to a bad override analyzer message in the internal repo. Maybe leave the parameter name as-is, and indicate with a comment that it is expected to be the device ip address.